src/components/MasterDetails.js

```diff
@@ -130,9 +130,9 @@ type MasterDetailsDefaultProps = {
 };
 
 type MasterDetailsState = {
-  /** The index for the most-recently selected row, to be displayed in the details pane. */
+  /** The data for the active row, to be displayed in the details pane. */
```

Comment:
I forgot to update this comment previously. This PR doesn't change the behavior, but makes the jsDoc accurate.